libs/external-db-firestore/tests/e2e-testkit/firestore_resources.ts

```diff
@@ -1,8 +1,9 @@
+export { supportedOperations } from '../../src/supported_operations'
```

Comment:
We already export it in `src/index.ts`, why export it twice?

Reply:
```
export interface IImplementationResources {
    initEnv(): Promise<void>
    shutdownEnv(): Promise<void> 
    setActive(): Promise<void> | void
    cleanup(): Promise<void> | void   
    supportedOperations: SchemaOperations[]
    name: string
}
```
E2EResources expects to receive implementation resources from the interface above.
Although it is exported elsewhere, it is important also for test resources and to enable initializing the constructor with one import instead of several